components/Common/Banner/index.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+import { ArrowUpRightIcon } from '@heroicons/react/24/outline';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type BannerProps = {
+  type: 'default' | 'error' | 'warning';
+  text: string;
+  url: string;
```

Comment:
I think that URL should be an optional argument. If there's no URL there should be no link

Reply:
So just display a text ?